xAPI.md

```diff
@@ -2483,8 +2483,18 @@ See <a href="#AppendixE">Appendix E: Example Signed Statement</a> for an example
 
 ## 3.0 Metadata
 
+Metadata is additional information about the resource.  It enables decision making, search, and discoverability within 
+systems.  In xAPI, metadata can be utilized in a variety of locations.  The most common is within <a name="actdef"> Activity Definitions </a>, 
+where the metadata describes the Activity. Most types of metadata have the opportunity to have Extensions.  
+Other uses include Verbs, Activity types, and attachments.
+
 ### 3.1 IRI Requirements
 
+Up-to-date information is the most effective. xAPI metadata is no exception, which is why xAPI uses IRIs for identifiers.
+Using IRIs ensures uniqueness and promotes resolvability is the IRI.  The LRS and Activity each have responsibilities 
+in regard to each IRI as outline below.  Activity Definitions have additional rules which can be found in <a name="actdef"> 
+this section</a>.
+
 ###### Metadata Requirements
```

Comment:
This part is in the wrong place which is leading to a confusing introduction above. Instead, section 3 should start with the text currently in 3.2. 

The current 3.1 is taken from the statements/activity section in the old ordering and is requirements specifically relating to activity IRIs. It should be titled accordingly and fitted into the current 3.2 rather than being it's own 3.x section. 

I'd delete this new 3.1 introduction and replace it with text specifically about activity ids.